addons/l10n_de/models/chart_template.py

```diff
@@ -12,6 +12,7 @@ def _get_de_res_company(self):
             self.env.company.id: {
                 'external_report_layout_id': 'l10n_din5008.external_layout_din5008',
                 'paperformat_id': 'l10n_din5008.paperformat_euro_din',
+                'check_account_audit_trail': True,
```

Comment:
Shouldn't it be in the new module?
If you install the new module, it's not checked. I don't know if it's intentional

Reply:
So, the logic, is that even if we don't have the new module, but that we have already audit trail installed, we want to activate it for new companies.